tests/test_examples.py

```diff
@@ -47,6 +49,16 @@ def test_examples(example_dir):
     actual_dir = example_dir.joinpath("output", "actual")
     dep_file_path = example_dir.joinpath("dependencies.yaml")
 
+    # Copy pyproject.toml files from expected to actual since they are modified in place
```

Comment:
Would it make sense to expect that if an output path is provided, that the `pyproject.toml` file is copied there?

Reply:
I'm not sure that I understand, could you elaborate?

The file tree under "actual" should match the file tree under "expected" exactly. If an output path is provided, then it is assumed that the file tree under "expected" will nest pyproject.toml at that path. Does that answer your question?

Reply:
Sorry, I misunderstood the purpose of lines 52-56. I was proposing to have a mechanism that would not actually modify the pyproject.toml in place, but write to a different location (in this case `actual`). But that probably does not make any sense outside of this particular test context.